src/helpers/find-titles.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+const findTitles = (titles) => {
```

Comment:
I believe this method is ok and works, just to let you know you can use `reduce` for the same result